src/gmp-test/ChainASwapCaller.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { IAxelarGateway } from '../interfaces/IAxelarGateway.sol';
+import { IAxelarGasReceiver } from '../interfaces/IAxelarGasReceiver.sol';
+import { IERC20 } from '../interfaces/IERC20.sol';
+
+contract ChainASwapCaller {
```

Comment:
If chain a refers to the source chain and chain b refers to the destination chain, then instead they should be named as such